### PR TITLE
chore(web): Update Terms and Conditions and Privacy Policy

### LINKED ISSUE #281: WEB: Update Terms and Condition, and Privacy Policy with attached markdown
## Description
Copy the contents of [`TERMS AND CONDITIONS.md`](https://github.com/user-attachments/files/15859823/TERMS.AND.CONDITIONS.md) and [`PRIVACY NOTICE.md`](https://github.com/user-attachments/files/15859845/PRIVACY.NOTICE.md) and paste it in [`apps/web/src/app/(main)/(mdx)/terms_and_condition/page.mdx`](https://github.com/keyshade-xyz/keyshade/blob/d82782f03c98514307a69e4ad4783cf9dfd48894/apps/web/src/app/(main)/(mdx)/terms_and_condition/page.mdx) and [`apps/web/src/app/(main)/(mdx)/privacy/page.mdx`](https://github.com/keyshade-xyz/keyshade/blob/d82782f03c98514307a69e4ad4783cf9dfd48894/apps/web/src/app/(main)/(mdx)/privacy/page.mdx)

### PR DESCRIPTION
**Pull request by Aleesssino**
## Description

- Updated Terms and Conditions and Privacy Policy documents to reflect current policies and legal requirements.
- Fixed style prop error in Privacy Policy markdown:
  Replace 
```
  <span style="text-decoration:underline;">Withdrawing your consent:</span>
```
  with 
```
   <span style={{ textDecoration: 'underline' }}>Withdrawing your consent:</span>
```
  to correctly apply inline style formatting using JSX. <br />

Fixes #281 

## Mentions

@kriptonian1 

## Developer's checklist

- [X] My PR follows the style guidelines of this project
- [X] I have performed a self-check on my work

**If changes are made in the code:**

- [X] I have followed the [coding guidelines](https://google.github.io/styleguide/jsguide.html)
- [X] My changes in code generate no new warnings
- [ ] My changes are breaking another fix/feature of the project
- [ ] I have added test cases to show that my feature works
- [ ] I have added relevant screenshots in my PR
- [X] There are no UI/UX issues

___

### **PR Type**
documentation, bug fix

___

### **Description**
- Updated Terms and Conditions and Privacy Policy documents to reflect current policies and legal requirements.
- Corrected markdown syntax for headers in Privacy Policy document.
- Fixed inline style formatting for JSX in Privacy Policy document.
- Improved formatting and readability in both documents.
- Added detailed sections on services, intellectual property, user responsibilities, and dispute resolution in Terms and Conditions document.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation
</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>page.mdx</strong><dd><code>Update Privacy Policy document with formatting fixes and style </code><br><code>corrections.</code></dd></summary>
<hr>

apps/web/src/app/(main)/(mdx)/privacy/page.mdx
<li>Corrected markdown syntax for headers.<br> <li> Fixed inline style formatting for JSX.<br> <li> Added missing punctuation and improved formatting.<br>

</details>
    

  </td>
  <td><a href="https://github.com/keyshade-xyz/keyshade/pull/282/files#diff-7377dd7e703eee6d7f836dff64fbd7ed22ee00e2924fe89ac771071813fe0b35">+41/-22</a>&nbsp; </td>
</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>page.mdx</strong><dd><code>Comprehensive update to Terms and Conditions document.</code>&nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

apps/web/src/app/(main)/(mdx)/terms_and_condition/page.mdx
<li>Updated terms and conditions to reflect current policies.<br> <li> Improved formatting and readability.<br> <li> Added detailed sections on services, intellectual property, user <br>responsibilities, and dispute resolution.<br>

</details>
    

  </td>
  <td><a href="https://github.com/keyshade-xyz/keyshade/pull/282/files#diff-b74b3c0a98f8466f92e0747acf832728e279e7d3f00ed269f054cb38460d6851">+488/-241</a></td>
</tr>                    
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**:
>Comment `/help` on the PR to get a list of all available PR-Agent tools and their descriptions

### CODE REVIEW

**Comment by rajdip-b** [comment]
@kriptonian1 yo ser drop your reviews!

**Review by rajdip-b — APPROVED** [review]
LGTM

File: apps/web/src/app/(main)/(mdx)/privacy/page.mdx [hunk 1]
```diff
@@ -18,96 +20,113 @@ Sensitive Information. We do not process sensitive information.
...
 
 We will use the information we receive only for the purposes that are described in this privacy notice or that are otherwise made clear to you on the relevant Services. Please note that we do not control, and are not responsible for, other uses of your personal information by your third-party social media provider. We recommend that you review their privacy notice to understand how they collect, use, and share your personal information, and how you can set your privacy preferences on their sites and apps.
 
-## 6\. IS YOUR INFORMATION TRANSFERRED INTERNATIONALLY?
+
+## 6. IS YOUR INFORMATION TRANSFERRED INTERNATIONALLY?
 
 _In Short: We may transfer, store, and process your information in countries other than your own._
 
 Our servers are located in. If you are accessing our Services from outside, please be aware that your information may be transferred to, stored, and processed by us in our facilities and by those third parties with whom we may share your personal information (see "WHEN AND WITH WHOM DO WE SHARE YOUR PERSONAL INFORMATION?" above), in and other countries.
```
Comment (kriptonian1) [inline apps/web/src/app/(main)/(mdx)/privacy/page.mdx:73]: update this too `Our servers are located in India. If you are accessing our Services from outside, please be aware that your information may be transferred to, stored, and processed by us in our facilities and by those third parties with whom we may share your personal information (see "WHEN AND WITH WHOM DO WE SHARE YOUR PERSONAL INFORMATION?" above), in India and other countries.`

File: apps/web/src/app/(main)/(mdx)/privacy/page.mdx [hunk 2]
```diff
@@ -18,96 +20,113 @@ Sensitive Information. We do not process sensitive information.
...
 
 When we have no ongoing legitimate business need to process your personal information, we will either delete or anonymize such information, or, if this is not possible (for example, because your personal information has been stored in backup archives), then we will securely store your personal information and isolate it from any further processing until deletion is possible.
 
-## 8\. DO WE COLLECT INFORMATION FROM MINORS?
+
+## 8. DO WE COLLECT INFORMATION FROM MINORS?
 
 _In Short: We do not knowingly collect data from or market to children under 18 years of age._
 
 We do not knowingly solicit data from or market to children under 18 years of age. By using the Services, you represent that you are at least 18 or that you are the parent or guardian of such a minor and consent to such minor dependent’s use of the Services. If we learn that personal information from users less than 18 years of age has been collected, we will deactivate the account and take reasonable measures to promptly delete such data from our records. If you become aware of any data we may have collected from children under age 18, please contact us at [support@keyshade.xyz](mailto:support@keyshade.xyz) .
```
Comment (kriptonian1) [inline apps/web/src/app/(main)/(mdx)/privacy/page.mdx:91]: Update this `We do not knowingly solicit data from or market to children under 18 years of age.` to `We do not knowingly solicit data from or market to individuals under the age of 18.`

File: apps/web/src/app/(main)/(mdx)/terms_and_condition/page.mdx [hunk 3]
```diff
@@ -1,299 +1,547 @@
 # TERMS AND CONDITIONS
 
-Last updated April 01, 2024
+
+Last updated: April 11, 2024.
```
Comment (kriptonian1) [inline apps/web/src/app/(main)/(mdx)/terms_and_condition/page.mdx:4]: Update the date to `11 April 2024`

**Review by kriptonian1 — CHANGES_REQUESTED** [review]
I think after this updates it should be good to merge

File: apps/web/src/app/(main)/(mdx)/terms_and_condition/page.mdx
Comment (Aleesssino) [inline apps/web/src/app/(main)/(mdx)/terms_and_condition/page.mdx:1]: I added some spacing for readability.

**Comment by rajdip-b** [comment]
@kriptonian1 can you review this now?

**Comment by kriptonian1** [comment]
Sorry guys, I was not available. Let me review it for you

File: apps/web/src/app/(main)/(mdx)/privacy/page.mdx [hunk 4]
```diff
@@ -18,96 +20,113 @@ Sensitive Information. We do not process sensitive information.
...
 We may update this privacy notice from time to time. The updated version will be indicated by an updated "Revised" date and the updated version will be effective as soon as it is accessible. If we make material changes to this privacy notice, we may notify you either by prominently posting a notice of such changes or by directly sending you a notification. We encourage you to review this privacy notice frequently to be informed of how we are protecting your information.
 
-## 12\. HOW CAN YOU CONTACT US ABOUT THIS NOTICE?
+
+## 12. HOW CAN YOU CONTACT US ABOUT THIS NOTICE?
 
 If you have questions or comments about this notice, you may contact us by post at:
 
-[support@keyshade.xyz](mailto:support@keyshade.xyz)
+support@keyshade.xyz
```
Comment (kriptonian1) [inline apps/web/src/app/(main)/(mdx)/privacy/page.mdx:127]: this should be `[support@keyshade.xyz](mailto:support@keyshade.xyz)`

File: apps/web/src/app/(main)/(mdx)/terms_and_condition/page.mdx [hunk 5]
```diff
@@ -1,299 +1,548 @@
...
+
+We are SAASONIC INNOVATIONS PRIVATE LIMITED ("Company," "we," "us," "our"), a company registered in India at 30/17 SHIB CHANDRA, CHATTERJEE STRT, Howrah, West Bengal 711202.
+
+ 
+
+We operate the website[ https://keyshade.xyz](https://keyshade.xyz/) (the "Site"), as well as any other related products and services that refer to or link to these legal terms (the "Legal Terms") (collectively, the "Services").
+
+ 
+
+You can contact us by phone at, by email at support@keyshade.xyz, or by mail at 30/17 SHIB CHANDRA CHATTERJEE STRT, Howrah, West Bengal 711202, India.
```
Comment (kriptonian1) [inline apps/web/src/app/(main)/(mdx)/terms_and_condition/page.mdx:20]: format this email as `[support@keyshade.xyz](mailto:support@keyshade.xyz)`

File: apps/web/src/app/(main)/(mdx)/terms_and_condition/page.mdx [hunk 6]
```diff
@@ -1,299 +1,548 @@
...
+
+ 
+
+These Legal Terms constitute a legally binding agreement made between you, whether personally or on behalf of an entity ("you"), and SAASONIC INNOVATIONS PRIVATE LIMITED, concerning your access to and use of the Services. You agree that by accessing the services, you have read, understood, and agreed to be bound by all of these legal terms. IF YOU DO NOT AGREE WITH ALL OF THESE LEGAL TERMS, THEN YOU ARE EXPRESSLY PROHIBITED FROM USING THE SERVICES AND SERVICES, AND YOU MUST DISCONTINUE USE IMMEDIATELY.
+
+ 
 
-We operate the website [https://keyshade.xyz](https://keyshade.xyz/) (the "Site"), as well as any other related products and services that refer or link to these legal terms (the "Legal Terms") (collectively, the "Services").
+We will provide you with prior notice of any scheduled changes to the services you are using. The modified legal terms will become effective upon posting or notifying you at support@keyshade.xyz, as stated in the email message. By continuing to use the services after the effective date of any changes, you agree to be bound by the modified terms.
```
Comment (kriptonian1) [inline apps/web/src/app/(main)/(mdx)/terms_and_condition/page.mdx:28]: format the email as `[support@keyshade.xyz](mailto:support@keyshade.xyz)`

File: apps/web/src/app/(main)/(mdx)/terms_and_condition/page.mdx [hunk 7]
```diff
@@ -1,299 +1,548 @@
...
 
-If you wish to make any use of the Services, Content, or Marks other than as set out in this section or elsewhere in our Legal Terms, please address your request to: [support@keyshade.xyz](mailto:support@keyshade.xyz). If we ever grant you the permission to post, reproduce, or publicly display any part of our Services or Content, you must identify us as the owners or licensors of the Services, Content, or Marks and ensure that any copyright or proprietary notice appears or is visible on posting, reproducing, or displaying our Content.
+ 
 
-We reserve all rights not expressly granted to you in and to the Services, Content, and Marks.
+If you wish to make any use of the Services, Content, or Marks other than as set out in this section or elsewhere in our Legal Terms, please address your request to: support@keyshade.xyz. If we ever grant you permission to post, reproduce, or publicly display any part of our services or content, you must identify us as the owners or licensors of the services, content, or marks and ensure that any copyright or proprietary notice appears or is visible when posting, reproducing, or displaying our content.
```
Comment (kriptonian1) [inline apps/web/src/app/(main)/(mdx)/terms_and_condition/page.mdx:87]: format the email as `[support@keyshade.xyz](mailto:support@keyshade.xyz)`

File: apps/web/src/app/(main)/(mdx)/terms_and_condition/page.mdx [hunk 8]
```diff
@@ -1,299 +1,548 @@
...
+
+ 
+
 
 ### Cancellation
 
-All purchases are non-refundable. You can cancel your subscription at any time by logging into your account.Your cancellation will take effect at the end of the current paid term. If you have any questions or are unsatisfied with our Services, please email us at [support@keyshade.xyz](mailto:support@keyshade.xyz).
+ 
+
+All purchases are non-refundable. You can cancel your subscription at any time by logging into your account. Your cancellation will take effect at the end of the current paid term. If you have any questions or are unsatisfied with our services, please email us at support@keyshade.xyz.
```
Comment (kriptonian1) [inline apps/web/src/app/(main)/(mdx)/terms_and_condition/page.mdx:216]: format the email to `[support@keyshade.xyz](mailto:support@keyshade.xyz)`

**Review by kriptonian1 — CHANGES_REQUESTED** [review]
There are just some formatting issues with the email, after fixing this we will be good to merge ig

**Review by kriptonian1 — APPROVED** [review]

**Review by rajdip-b — APPROVED** [review]

**Pull request merged**